### PR TITLE
Bug 1477945 Environment and cache parameters added to subsystem=naming/binding fo…

### PR DESCRIPTION
**Pull request by rubenvp8510**
…r EAP6

### CODE REVIEW

File: modules/plugins/jboss-as-7/src/main/resources/META-INF/rhq-plugin.xml [hunk 1]
```diff
@@ -5922,6 +5929,13 @@
           <c:simple-property name="module" required="false" type="string" readOnly="false" description="The module to load the object factory from for object factory bindings"/>
           <c:simple-property name="type" required="false" type="string" readOnly="false" description="The type of the value to bind for simple bindings, this must be a primitive type"/>
           <c:simple-property name="value" required="false" type="string" readOnly="false" description="The value to bind for simple bindings"/>
+          <c:simple-property name="cache" required="false" type="boolean" readOnly="false" defaultValue="false" description="If the external context should be cached"/>
+          <c:list-property name="environment" readOnly="true" description="A list of The environment to use on object factory instance retrieval." required="false">
+            <c:map-property name="environment" readOnly="true" displayName="Environment">
```
Comment (josejulio) [inline modules/plugins/jboss-as-7/src/main/resources/META-INF/rhq-plugin.xml]: why `readOnly=true` on the list and the map?

Comment (rubenvp8510) [reply modules/plugins/jboss-as-7/src/main/resources/META-INF/rhq-plugin.xml]: My mistake, should be readonly= false,

**Comment by rubenvp8510** [comment]
Updated with the observations, @josejulio could you do another quick review, please? Thanks

**Comment by josejulio** [comment]
LGTM

**Pull request merged**